src/content/configuration/optimization.md

```diff
@@ -324,4 +324,37 @@ module.exports = {
     concatenateModules: true
   }
 };
-```
\ No newline at end of file
+```
+
+## `optimization.sideEffects`
+
+`bool`
+
+Tells webpack to recognise the [`sideEffects`](https://github.com/webpack/webpack/blob/master/examples/side-effects/README.md) flag in `package.json` or rules to skip over modules which are flagged to contain no side effects when exports are not used. 
+
+__package.json__
+
+``` json
+{
+  "name": "awesome npm module",
+  "version": "1.0.0",
+  "sideEffects": false
+}
+```
+
+T> Please note that `sideEffects` should be in the big module's `package.json` file and doesn't mean that you need to set `sideEffects` to `false` in your own project's `package.json` which requires that big module.
```

Comment:
What is big module? Do you mean: main app?

Reply:
it means an npm module required by your project, follow the `sideEffects` url for more context

Reply:
maybe change it to:

Please note that `sideEffects` should be in the npm module's `package.json` file

Reply:
Yeah, that sounds better

Reply:
@montogeek done

Reply:
Perhaps a link to each [library devs](https://github.com/webpack/webpack/tree/master/examples/side-effects) and [app devs](https://webpack.js.org/guides/tree-shaking/) could be added?